template.distillery/Makefile.mobile

```diff
@@ -291,7 +290,7 @@ $(ADD_PLATFORMS): add-%: $(CORDOVAPATH)/platforms/%
 PLATFORMS := android ios browser
 
 # before building, environment variables are checked and assets are copied.
-$(PLATFORMS): %: check-app-env icons-% assets icons-cp-% add-%
+$(PLATFORMS): %: byte check-app-env icons-% assets icons-cp-% add-%
```

Comment:
`make run-android` does not work before `make byte`.
I will add it for emulate too.